inc/3rd-party/hosting/presslabs.php

```diff
@@ -0,0 +1,139 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
+
+if ( defined( 'PL_INSTANCE_REF' ) && class_exists( '\Presslabs\Cache\CacheHandler' ) ) {
+
+	add_action( 'pl_pre_cache_refresh', 'rocket_clean_post', 0 );
```

Comment:
You should be using `rocket_clean_files` instead

Reply:
👍 change done!